js/latoken.js

```diff
@@ -164,8 +164,10 @@ module.exports = class latoken extends Exchange {
             const base = this.safeCurrencyCode (baseId);
             const quote = this.safeCurrencyCode (quoteId);
             const symbol = base + '/' + quote;
+            const pricePrecisionString = this.safeString (market, 'pricePrecision');
+            const pricePrecision = (pricePrecisionString === undefined) ? undefined : '1e-' + pricePrecisionString;
             const precision = {
-                'price': this.safeInteger (market, 'pricePrecision'),
+                'price': parseInt (pricePrecisionString),
```

Comment:
There's at least two problems:
1. we `parseInt` a possibly-undefined value that will throw an uncaught exception in js
2. 
```
~/ccxt $ node
Welcome to Node.js v14.15.4.
Type ".help" for more information.
> const pricePrecisionString = '8'
undefined
> parseInt ('1e-' + pricePrecisionString)
1
```

Reply:
Gotta refix zb and gemini: https://github.com/ccxt/ccxt/pull/8979

Reply:
if you look carefully we are doing `parseInt (pricePrecisionString)` and not `parseInt (pricePrecision)`, so it works as intended.

> we parseInt a possibly-undefined value that will throw an uncaught exception in js

I don't think this is so critical because it is in fetchMarkets so it can be easily resolved, moreover it will throw an error in python and let us know it is broken in the travis. finally the alternative code was:

```
'price': {
     'min': Math.pow (10, -precision['price']),
     'max': undefined,
 },
 ```
 
 which also throws an error if precision is undefined!

Reply:
@frosty00 the problem is that `1e-8` will yield a precision of 1, and this value does not look right to me...

```
> const pricePrecisionString = '8'
undefined
> parseInt ('1e-' + pricePrecisionString)
1
```

Reply:
> parseInt ('1e-' + pricePrecisionString)

right but we never do this anywhere in the lib, we just do `parseFloat ('1e-' + pricePrecisionString)

Reply:
We do parseInt ('1e' + pricePrecisionString) in this PR (the line above this thread) and in zb, which is wrong, no?

Reply:
Pardon, actually, it looks like I misread it.